<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QDialog" name="EditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonsLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/img/wrong.png</normaloff>:/img/img/wrong.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/img/correct.png</normaloff>:/img/img/correct.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>53</y>
     <width>802</width>
     <height>726</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="editpic">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="img.qrc">:/img/img/username.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="editUserMainLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="labelsLayout">
        <item>
         <widget class="QLabel" name="firstNameLabel">
          <property name="text">
           <string>First Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lastNameLabel">
          <property name="text">
           <string>Last Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="userNameLabel">
          <property name="text">
           <string>Username: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="roleLabel">
          <property name="text">
           <string>Role: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="inputsLayout">
        <item>
         <widget class="QLineEdit" name="firstNameInput"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameInput"/>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameInput"/>
        </item>
        <item>
         <widget class="QComboBox" name="roleComboBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>Doctor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nurse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Administrator</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>editpic</zorder>
   <zorder>editUserMainLabel</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="firstNameEmptyLabel">
      <property name="text">
       <string>Empty!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lastNameEmptyLabel">
      <property name="text">
       <string>Empty!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="usernameNotValidLabel">
      <property name="text">
       <string>Not Valid!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
